src/components/Events.module.scss

```diff
@@ -15,8 +17,22 @@
   width: 100%;
   gap: 30px;
   justify-content: space-between;
+  max-width: 100%;
 }
 
 .sport {
   font-weight: bold;
 }
+
+@media only screen and (max-width: $screen-md-min) {
+  .event {
+    margin-left: auto;
+    margin-right: auto;
+    height: 200px;
+    width: 343px;
+    border: 2px solid #333;
+    padding-top: 5px;
+    padding-left: 10px;
+    border-radius: 10px;
```

Comment:
I can imagine some of these paddings make sense even for below-md screen. Look at this:

![CleanShot 2024-03-04 at 17 32 13](https://github.com/GLObus303/road-to-star/assets/12488648/01c0287b-5076-4c04-9408-336439bea45c)